client/src/components/shared/Info/Info.js

```diff
@@ -84,7 +84,11 @@ const Info = ({data, canClose, onClose}) => {
         <h3 className={`${isPolitician(data) ? 'politician' : ''}`}>
           <Circle aria-hidden="true" />&nbsp;{entity.entity_name}&nbsp;
         </h3>
-        <Link to={`/verejne?lat=${entity.lat}&lng=${entity.lng}&zoom=18`} title="Zobraz na mape">
+        <Link
+          to={`/verejne?lat=${entity.lat}&lng=${entity.lng}&zoom=18`}
+          title="Zobraz na mape"
+          onClick={window.location.reload}
```

Comment:
Reloading the window is not a good idea - for example when I use "Satelite" view map, clicking on MapMarker reloads the window and I'm in "Map" view again.

The old solution (https://beta.verejne.digital) used `showOnMap` to seamlessly navigate without refresh.
See `showOnMap` removed here: https://github.com/verejnedigital/verejne.digital/commit/3a43804eb49b63bb8460c0718593eb597bc9c527